src/Resources/Locales/de_DE.axaml

```diff
@@ -286,9 +286,9 @@
   <x:String x:Key="Text.GitLFS.Push.Title" xml:space="preserve">LFS Objekte pushen</x:String>
   <x:String x:Key="Text.GitLFS.Push.Tips" xml:space="preserve">Pushe große Dateien in der Warteschlange zum Git LFS Endpunkt</x:String>
   <x:String x:Key="Text.GitLFS.Remote" xml:space="preserve">Remote:</x:String>
-  <x:String x:Key="Text.GitLFS.Track" xml:space="preserve">Verfolge Dateien names '{0}'</x:String>
+  <x:String x:Key="Text.GitLFS.Track" xml:space="preserve">Verfolge '{0}' benannte Dateien</x:String>
```

Comment:
I'm not sold on this one, but I can see potential for improvement for my old translation.

Maybe just "Verfolge '{0}' Dateien" which would read for example "Verfolge 'bla.zip' Dateien".
Or "Verfolge _alle_ '{0}' Dateien" to emphasize that this is about all files named like this.

Reply:
"Verfolge alle '{0}' Dateien is a good one. 👍